Hermes/appscale/hermes/helper.py

```diff
@@ -222,11 +224,11 @@ def create_br_json_data(role, type, bucket_name, index, storage):
   if role == 'db_master':
     data[JSONTags.TYPE] = 'cassandra_{0}'.format(type)
     data[JSONTags.OBJECT_NAME] = "gs://{0}{1}".format(bucket_name,
-      hermes_constants.DB_MASTER_OBJECT_NAME)
+                                                      constants.DB_MASTER_OBJECT_NAME)
```

Comment:
Could you indent the format line to the next line so that its within the 80 limit?